packages/core/src/hooks.ts

```diff
@@ -51,6 +51,8 @@ export function initializeConnector<T extends Connector>(
 }
 
 function computeIsActive({ chainId, accounts, activating }: Web3ReactState) {
+  // accounts = [] indicates successful Infura connection
```

Comment:
Maybe we should just check `typeof accounts !== 'undefined'` so we don't need the comment?

Reply:
or check if it's an array i guess also work